### PR TITLE
Support spaces in build.sh paths

### PR DESCRIPTION
**Pull request by jolorunyomi**
# Summary
Add support for space in workspace

### CODE REVIEW

File: ci/docs/build.sh [hunk 1]
```diff
@@ -44,18 +44,18 @@ conda list --show-channel-urls
...
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/"$PROJECT/$BRANCH_VERSION/*	
 done
 
-mv $PROJECT_WORKSPACE/docs/build/html/* $DOCS_WORKSPACE/api/cuspatial/$BRANCH_VERSION
+mv "$PROJECT_WORKSPACE/docs/build/html/"* "$DOCS_WORKSPACE/api/cuspatial/"$BRANCH_VERSION
```
Comment (ajschmidt8) [inline ci/docs/build.sh:60]:
```suggestion
mv "$PROJECT_WORKSPACE/docs/build/html/"* "$DOCS_WORKSPACE/api/cuspatial/$BRANCH_VERSION"
```

I don't think there's any harm in moving this to the end of the line

File: ci/docs/build.sh [hunk 2]
```diff
@@ -44,18 +44,18 @@ conda list --show-channel-urls
...
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/"$PROJECT/$BRANCH_VERSION/*
```
Comment (ajschmidt8) [inline ci/docs/build.sh:57]:
```suggestion
    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/"*
```

**Review by ajschmidt8 — APPROVED** [review]
just a few stylistic suggestions here to move some terminating quotes to the end of the line, but should be fine either way.

**Comment by trxcllnt** [comment]
rerun tests

File: ci/gpu/build.sh [hunk 3]
```diff
@@ -102,13 +102,13 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
...
 
         #Python Unit tests for cuSpatial
-        cd $WORKSPACE/python/cuspatial
-        py.test --cache-clear --junitxml=${WORKSPACE}/junit-cuspatial.xml -v
+        cd "$WORKSPACE/python/cuspatial"
+        py.test --cache-clear --junitxml="$WORKSPACE/junit-cuspatial.xml" -v
     fi
 else
-    export LD_LIBRARY_PATH="$WORKSPACE/ci/artifacts/cuspatial/cpu/conda_work/cpp/build:$CONDA_PREFIX/lib:$LD_LIBRARY_PATH"
+    export LD_LIBRARY_PATH="$WORKSPACE/ci/artifacts/cuspatial/cpu/conda_work/cpp/build":$CONDA_PREFIX/lib:$LD_LIBRARY_PATH"
```
Comment (trxcllnt) [inline ci/gpu/build.sh:109]:
```suggestion
    export LD_LIBRARY_PATH="$WORKSPACE/ci/artifacts/cuspatial/cpu/conda_work/cpp/build:$CONDA_PREFIX/lib:$LD_LIBRARY_PATH"
```

**Review by trxcllnt — CHANGES_REQUESTED** [review]

File: ci/local/build.sh [hunk 4]
```diff
@@ -82,7 +82,7 @@ set -e
 WORKSPACE=${REPO_PATH_IN_CONTAINER}
 PREBUILD_SCRIPT=${REPO_PATH_IN_CONTAINER}/ci/gpu/prebuild.sh
 BUILD_SCRIPT=${REPO_PATH_IN_CONTAINER}/ci/gpu/build.sh
-cd \${WORKSPACE}
+cd "\$WORKSPACE"
```
Comment (trxcllnt) [inline ci/local/build.sh:85]:
```suggestion
cd \"\$WORKSPACE\"
```

**Comment by harrism** [comment]
@jolorunyomi can you respond to @trxcllnt 's review? Thanks.

**Comment by trxcllnt** [comment]
rerun tests

**Comment by harrism** [comment]
rerun tests

File: ci/cpu/build.sh [hunk 5]
```diff
@@ -88,8 +88,8 @@ if [ "$BUILD_CUSPATIAL" == '1' ]; then
   if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
     gpuci_conda_retry build --croot ${CONDA_BLD_DIR} conda/recipes/cuspatial
   else
-    gpuci_conda_retry build --croot ${CONDA_BLD_DIR} --dirty --no-remove-work-dir \
-        -c $WORKSPACE/ci/artifacts/cuspatial/cpu/.conda-bld/ conda/recipes/cuspatial
+    gpuci_conda_retry build --croot "$CONDA_BLD_DIR" --dirty --no-remove-work-dir \
+        -c "$WORKSPACE/ci/artifacts/cuspatial/cpu/conda-bld/" conda/recipes/cuspatial
```
Comment (ajschmidt8) [inline ci/cpu/build.sh:92]:
```suggestion
        -c "$WORKSPACE/ci/artifacts/cuspatial/cpu/.conda-bld/" conda/recipes/cuspatial
```

looks like a `.` was mistakenly removed here.

**Comment by ajschmidt8** [comment]
should be good to merge this pending @trxcllnt's re-review

**Review by trxcllnt — APPROVED** [review]

**Comment by harrism** [comment]
@gpucibot merge

**Pull request merged**